<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-25T23:54:23 PDT" name="Default suite" finished-at="2022-05-25T23:55:55 PDT" duration-ms="91478">
    <groups>
    </groups>
    <test started-at="2022-05-25T23:54:23 PDT" name="Default test" finished-at="2022-05-25T23:55:55 PDT" duration-ms="91478">
      <class name="testmyntra.AmazonWorking1$AmazonHeadersTestClass">
        <test-method is-config="true" signature="LounchBrouser()[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e]" started-at="2022-05-25T23:54:23 PDT" name="LounchBrouser" finished-at="2022-05-25T23:55:09 PDT" duration-ms="45945" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LounchBrouser -->
        <test-method is-config="true" signature="loginUserAmazon()[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e]" started-at="2022-05-25T23:55:09 PDT" name="loginUserAmazon" finished-at="2022-05-25T23:55:21 PDT" duration-ms="11578" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginUserAmazon -->
        <test-method signature="ClickOnFresh()[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e]" started-at="2022-05-25T23:55:21 PDT" name="ClickOnFresh" finished-at="2022-05-25T23:55:52 PDT" duration-ms="31180" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Amazon.in] but found [Amazon.in: Amazon Fresh]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Amazon.in] but found [Amazon.in: Amazon Fresh]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testmyntra.AmazonWorking1$AmazonHeadersTestClass.ClickOnFresh(AmazonWorking1.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ClickOnFresh -->
        <test-method is-config="true" signature="logoutFromApplicationTestFail(org.testng.ITestResult)[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e]" started-at="2022-05-25T23:55:52 PDT" name="logoutFromApplicationTestFail" finished-at="2022-05-25T23:55:54 PDT" duration-ms="1756" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=ClickOnFresh status=FAILURE method=AmazonWorking1$AmazonHeadersTestClass.ClickOnFresh()[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplicationTestFail -->
        <test-method is-config="true" signature="clearPOMobjects()[pri:0, instance:testmyntra.AmazonWorking1$AmazonHeadersTestClass@1e683a3e]" started-at="2022-05-25T23:55:54 PDT" name="clearPOMobjects" finished-at="2022-05-25T23:55:55 PDT" duration-ms="922" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearPOMobjects -->
      </class> <!-- testmyntra.AmazonWorking1$AmazonHeadersTestClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
